model/office/open.go

```diff
@@ -65,16 +65,18 @@ func Open(inst *instance.Instance, fileID string) (*Opener, error) {
 	if err != nil {
 		return nil, err
 	}
-	// TODO check that the file is an office document
-	return &Opener{inst: inst, file: file}, nil
+	if file.Class != "text" && file.Class != "spreadsheet" && file.Class != "slide" {
```

Comment:
@JF-Cozy I think this is what we said, but we need to have the same check on the front end side.

Reply:
Est-ce qu'on est vraiment pertinent avec ce test ? Par exemple un `.md` est un document de class `text` mais n'est pas pris en charge par onlyoffice. Ici on a forcément dans un context d'un doc pris en charge par onlyoffice ?

Reply:
Do you have better to suggest?

For a markdown file, it should be tested but I think it will be opened like a `.txt` one (no support for bold or italic, just the raw text).

Reply:
Je ne sais pas, il faudrait effectivement tester, on tombe peut-être sur une erreur car le fichier n'est pas pris en charge. Dans ce cas pas d'autre choix que de se baser sur les extensions non ?

Reply:
A l'ouverture d'un `.txt` ou `.md` avec onlyoffice on obtient une petite popup d'import du fichier. Une fois validé, on a bien le document ouvert dans l'éditeur : 

![image](https://user-images.githubusercontent.com/67680939/114536836-c5380900-9c51-11eb-8f6e-25cf91f90c03.png)

Reply:
OK, donc je laisse comme ça côté stack, et je vous laisse voir côté front dans quels cas il vaut mieux passer par OnlyOffice et dans quels cas on voudra privilégier quelque chose de plus léger.